features/guidance/anticipate-lanes.feature

```diff
@@ -151,9 +151,9 @@ Feature: Turn Lane Guidance
             | cj    |                    | 1     | motorway_link | yes    | xbcj |
 
        When I route I should get
-            | waypoints | route        | turns                           | lanes                          |
-            | a,i       | ab,ci,ci     | depart,turn slight right,arrive | ,none:false slight right:true, |
-            | a,j       | ab,xbcj      | depart,arrive                   | ,                              |
+            | waypoints | route        | turns                           | lanes                           |
+            | a,i       | ab,ci,ci     | depart,turn slight right,arrive | ;,none:false slight right:true, |
+            | a,j       | ab,xbcj      | depart,arrive                   | ;;none:true slight right:false, |
```

Comment:
Why do lanes change here? Lanes should not be affected by the UseLane removal, no?